src/lib/errors/Identifiers.ts

```diff
@@ -40,21 +40,13 @@ export const enum Identifiers {
 
 	CommandDisabled = 'commandDisabled',
 
-	PreconditionCooldown = 'preconditionCooldown',
-	PreconditionDMOnly = 'preconditionDmOnly',
-	PreconditionGuildNewsOnly = 'preconditionGuildNewsOnly',
-	PreconditionGuildNewsThreadOnly = 'preconditionGuildNewsThreadOnly',
-	PreconditionGuildOnly = 'preconditionGuildOnly',
-	PreconditionGuildPrivateThreadOnly = 'preconditionGuildPrivateThreadOnly',
-	PreconditionGuildPublicThreadOnly = 'preconditionGuildPublicThreadOnly',
-	PreconditionGuildTextOnly = 'preconditionGuildTextOnly',
 	PreconditionNSFW = 'preconditionNsfw',
 	PreconditionClientPermissions = 'preconditionClientPermissions',
 	PreconditionClientPermissionsNoClient = 'preconditionClientPermissionsNoClient',
 	PreconditionClientPermissionsNoPermissions = 'preconditionClientPermissionsNoPermissions',
 	PreconditionUserPermissions = 'preconditionUserPermissions',
 	PreconditionUserPermissionsNoPermissions = 'preconditionUserPermissionsNoPermissions',
-	PreconditionThreadOnly = 'preconditionThreadOnly',
+  PreconditionChannelType = 'preconditionChannelType',
```

Comment:
```suggestion
	PreconditionChannelType = 'preconditionChannelType',
```

Run `yarn lint` or `yarn format` or make sure husky sets up the git hooks. You can check our contributing guides for more information: https://github.com/sapphiredev/.github/blob/main/.github/CONTRIBUTING.md